DocumentDataAPI/DocumentDataAPI/Controllers/WordRatioController.cs

```diff
@@ -0,0 +1,191 @@
+using DocumentDataAPI.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DocumentDataAPI.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class WordRatioController : ControllerBase
+{
+    private readonly IWordRatioRepository _repository;
+
+    public WordRatioController(IWordRatioRepository repository)
+    {
+        _repository = repository;
+    }
+
+    [HttpGet]
+    [Route("GetAll/")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public ActionResult<IEnumerable<WordRatioModel>> GetAll()
+    {
+        try
+        {
+            IEnumerable<WordRatioModel> result = _repository.GetAll();
+            return result.Any()
+                ? Ok(result)
+                : NoContent();
+        }
+        catch (DbException e)
+        {
+            return Problem(e.Message);
+        }
+    }
+
+
+    [HttpGet]
+    [Route("GetByDocumentIDAndWord/{id:int}/{word}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public ActionResult<WordRatioModel> GetByDocumentIdAndWord(int documentId, string word)
+    {
+        try
+        {
+            WordRatioModel? result = _repository.GetByDocumentIdAndWord(documentId, word);
+            return result != null
+                ? Ok(result)
+                : NoContent();
+        }
+        catch (DbException e)
+        {
+            return Problem(e.Message);
+        }
+    }
+
+    [HttpGet]
+    [Route("GetByDocumentId/{id:int}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public ActionResult<IEnumerable<WordRatioModel>> GetByDocumentId(int documentId)
+    {
+        try
+        {
+            IEnumerable<WordRatioModel> result = _repository.GetByDocumentId(documentId);
+            return result.Any()
+                ? Ok(result)
+                : NoContent();
+        }
+        catch (DbException e)
+        {
+            return Problem(e.Message);
+        }
+    }
+
+    [HttpGet]
+    [Route("GetByWord/{word}")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public ActionResult<IEnumerable<WordRatioModel>> GetByWord(string word)
+    {
+        try
+        {
+            IEnumerable<WordRatioModel> result = _repository.GetByWord(word);
+            return result.Any()
+                ? Ok(result)
+                : NoContent();
+        }
+        catch (DbException e)
+        {
+            return Problem(e.Message);
+        }
+    }
+
+    [HttpGet]
+    [Route("GetByWords/{wordlist}")]
```

Comment:
mangler der ikke "ProduceResponseType" på en del endpoints her?
```suggestion
    [Route("GetByWords/{wordlist}")]
    [ProducesResponseType(StatusCodes.Status200OK)]
    [ProducesResponseType(StatusCodes.Status204NoContent)]
    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```

Reply:
Jo, det tilføjer jeg lige!